dist/tools/pic32prog/doc.md

```diff
@@ -0,0 +1,136 @@
+Flashing on Linux using pickit-3
+================================
+
+Allow flashing a pic32-wifire from linux using https://github.com/sergev/pic32prog
+
+
+Informations come from this comment
+
+https://github.com/RIOT-OS/RIOT/pull/6092#issuecomment-261987955
+
+
+Steps
+-----
+
+### Setup your computer
+
+I ran this on Ubuntu 16.04
+
+* Install Virtualbox
+* Add yourself to the 'vboxusers' group
+  `sudo usermod -a -G vboxusers your_username`
+* Reboot (was necessary for the ext-pack may be possible without)
+
+### Setup Windows VM
+
+You can download a free 90 days valid windows virtualbox image from:
+
+    https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/
+
+I downloaded a Windows 10 virtualbox image
+
+* Create a machine in virtualbox in File-Import Appliance.
+  * optional: disabled network
+* Start and login with password: Passw0rd!
+* optional: set an empty password in Settings-Accounts
+* optional: Create a shared folder with your host machine. I run an offline
+  windows machine so download everything from my host.
+* Install Virtualbox Guest-additions for USB support (reboot)
+* USB add both 'microchip' usb as USB1 devices
+
+### Downloads
+
+Download and extract:
+
+* PICkit3-Programmer http://ww1.microchip.com/downloads/en/DeviceDoc/PICkit3%20Programmer%20Application%20v3.10.zip
+
+Adapted from the README (on Windows 10 machine .NET 4.00 is already installed)
+
+* Extract `PICkit3 Programmer Application Setup v3.10.zip`
+* Run `setup.exe`
+* Run PICkit3
+* Tools/Download PICkit Operating System
+  * Select c://Program Files/Microchip/PICkit3/PK3OSV020005.hex
+* Device should be flashed there will be a message saying the board is found
+* Turn off your VM
```

Comment:
The application is crashing when exiting. Maybe calm the reader down, that this is "normal" ;-).